Quadriga/src/main/java/edu/asu/spring/quadriga/web/config/SecurityContext.java

```diff
@@ -0,0 +1,171 @@
+                            "/public/**", "/search/**", "/home/**", "/sociallogin*", "/forbidden")
+                    .permitAll()
+                    // The rest of the our application is protected.
+                    .antMatchers("/users/**", "/admin/**").hasRole("QUADRIGA_USER_ADMIN")
+                    .antMatchers("/", "/auth/welcome**", "/auth/home**", "/auth/about**", "/auth/profile/**",
+                            "/rest/**", "/forbidden")
+                    .hasAnyRole("QUADRIGA_USER_ADMIN", "QUADRIGA_USER_STANDARD", "QUADRIGA_USER_COLLABORATOR")
+                    .antMatchers("/auth/workbench/**", "/auth/rest/**", "/auth/conceptcollections/**",
+                            "/auth/conceptdetails/**", "/auth/transformation/**", "/auth/searchitems/**",
+                            "/auth/dictionaries/**", "/auth/editing/**", "/auth/networks/**")
+                    .hasAnyRole("QUADRIGA_USER_STANDARD", "QUADRIGA_USER_COLLABORATOR")
+                    .antMatchers("/users/**", "/admin/**", "/auth/users/**", "/checks/**")
+                    .hasRole("QUADRIGA_USER_ADMIN").anyRequest()
+                    .hasAnyRole("QUADRIGA_USER_ADMIN", "QUADRIGA_USER_STANDARD", "QUADRIGA_USER_COLLABORATOR")
+                    .antMatchers("/**").denyAll()
+                    // Adds the SocialAuthenticationFilter to Spring Security's
+                    // filter chain.
+                    .and().apply(new SpringSocialConfigurer());
+
+        }
+
+        @Autowired
+        @Override
+        protected void configure(AuthenticationManagerBuilder auth) throws Exception {
+            auth.authenticationProvider(daoAuthProvider());
+            auth.eraseCredentials(false);
+            auth.inMemoryAuthentication().withUser("admin").password("admin").roles("QUADRIGA_USER_ADMIN");
+            super.configure(auth);
+
+        }
+
+        @Bean(name = BeanIds.AUTHENTICATION_MANAGER)
+        @Override
+        public AuthenticationManager authenticationManagerBean() throws Exception {
+            return super.authenticationManager();
+        }
+
+        @Bean
+        public PasswordEncoder passwordEncoder() {
+            return new BCryptPasswordEncoder(10);
+        }
+
+        @Bean
+        public SocialUserDetailsService socialUserDetailsService() {
+            return new SimpleSocialUserDetailsService(userDetailsService());
+        }
+
+        @Bean
+        public UserDetailsService userDetailsService() {
+            return userService;
+        }
+
+        @Bean
+        public DaoAuthenticationProvider daoAuthProvider() throws Exception {
+            DaoAuthenticationProvider provider = new DaoAuthenticationProvider();
+            provider.setPasswordEncoder(passwordEncoder());
+            provider.setUserDetailsService(userDetailsService());
+            return provider;
+        }
+        
+        @Bean
+        public AccessDeniedHandler accessDeniedHandler(){
```

Comment:
the user should be redirected to the login page if they are not logged in but try to access anything in /auth

Reply:
Yes. The user is redirected to login page if they are not logged in and try to access anything in /auth